iai_donbot_description/robots/iai_donbot.urdf.xacro

```diff
@@ -107,7 +107,13 @@
 			<child link="camera_holder_link" />
 			<origin xyz="0 0 0" rpy="0 0 ${pi}" />
 		</joint>
-		<link name="camera_link" />
+		<link name="camera_link">
+			<visual>
+				<geometry>
+					<cylinder radius="0.001" length="0.005"/>
+				</geometry>
+			</visual>
+		</link>
```

Comment:
Why have you added a small cylinder to the camera link?

Reply:
It's for unreal to recognize this link, link without visual or collision will be ignored. I need this link for camera frame